src/layer/marker/Marker.Label.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Label extension to L.Marker, adding label-related methods.
+ */
+
+L.Marker.include({
+	_getLabelAnchor: function () {
+		return this.options.icon.options.labelAnchor || [0, 0];
+	}
+});

```

Comment:
Or merge with `Marker.Popup.js` in someting like `Marker.PopupBase.js` (whatever the name we'll chose? Just for consistency between the two.